test/obb/events/game_events_test.clj

```diff
@@ -21,29 +21,20 @@
   (let [system (-> (game-events-system) component/start)
         game-events (:game-events system)
         game-info (game-info/create)
-        subscriber-ch (async/chan 1)
+        game-ch (async/chan 1)
         subscriber-all-ch (async/chan 1)
         test-event :new-game
-        expected-message {:event-name test-event
-                          :game-id (game-info/id game-info)
+        expected-message {:game-id (game-info/id game-info)
                           :data game-info}]
```

Comment:
After reading the next commit, I realized that the event name is also a nice candidate to the metadata field.